src/SharedAcceptanceTests.RequirePartitionKey/When_using_outbox_synchronized_session_via_container.cs

```diff
@@ -34,14 +34,10 @@ public Endpoint()
                 EndpointSetup<DefaultServer>(config =>
                 {
                     config.EnableOutbox();
+                    config.UsePersistence<CosmosDbPersistence>().RegisterSharedTransactionalBatchForDependencyInjection();
```

Comment:
@danielmarbach, there was a way to access the currently used persistence w/o calling `UsePersistence<>` again. What was it?